scripts/build-rancherd

```diff
@@ -14,11 +19,14 @@ if [ ! -z $DRONE_BRANCH ] && [ -z $DRONE_TAG ];then
 fi
 echo "RUNTIME_TAG: $RUNTIME_TAG"
 
-K8S_VERSION=$(grep 'ARG RKE2_VERSION=' ../../package/Dockerfile.runtime | sed 's/.*RKE2_VERSION=//')
 go build -o ../../bin/rancherd-${ARCH} \
     -ldflags '-w -s -extldflags "-static"
-    -X github.com/rancher/rke2/pkg/images.KubernetesVersion='$K8S_VERSION'-'$ARCH'
+    -X github.com/rancher/rke2/pkg/images.KubernetesVersion='${K8S_VERSION}'-'${ARCH}'
     -X github.com/rancher/rke2/pkg/images.RuntimeImageName=rancher-runtime
+    -X github.com/rancher/rke2/pkg/images.DefaultEtcdImage=rancher/hardened-etcd:'${ETCD_VERSION}'-'${IMAGE_BUILD_VERSION}'
+    -X github.com/rancher/rke2/pkg/images.DefaultKubernetesImage='${REPO}'/hardened-kubernetes:'${K8S_VERSION}'
```

Comment:
Can REPO be anything other than rancher during the build process?

Reply:
No, I will update.

Reply:
It can be for devs when building RKE2 locally and pushing to a different docker hub org, but not for y'all.